### PR TITLE
Updates and Fixes Gas Overlays and Icy Floor Change

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Updates the gas sprites for plasma and N2O/sleeping gas to what TG has in addition to updating the overlay system to work like theirs does, minus the alpha setting thing. Also removes the icy overlay (will explain in a moment).

When we got updated gas sprites, clicking on things became difficult because they took up the entire turf. There was a hacky fix put in place that allowed you to pick up items, and such, but none the less generated a bunch of new problems (like not being able to right-click/view the turf itself if plasma/sleeping gas was on it).

fully and properly fixes: https://github.com/ParadiseSS13/Paradise/issues/5195
hopefully fixes: https://github.com/ParadiseSS13/Paradise/issues/10794

Icy turfs have also been changes. Cold air no longer makes turfs icy, directly. Instead, cryostylane, applied to a turf, makes it icy; you'll have to wear footwear that prevents slipping to not slip on ice. If you're not careful, you could bash your head too.

Having there be non slippery and slippery iced turfs would be confusing to players, and making LINDA make turfs *actually* slippery would be hilariously broken, especially since it already throws and blenders people an awful lot, already.

Hopefully people will actually bother to look at air alarms now and go by their atmospheric scan on their PDA (everyone has one!).

Pics of the new gas sprites

img
Plasma looks a lot more like it used to, while being obviously updated
img
More whispy

:cl: Fox McCloud
tweak: updates plasma and N2O sprites
fix: Fixes not being able to look at a turf if it had plasma/N2O on it
del: cold turfs no longer become icy
add: Cryostylane makes turfs icy
add: Icy turfs are slippery; wear magboots to counteract the effect. Watch your head!
fix: Fixes grammar of slipping on turfs; no more "you slipped on floor" or "you slipped on wet floor"
/:cl:

### CODE REVIEW

**Comment by Mitchs98** [comment]
Kinda dislike the removal of icy turfs. You'd be really unable at all to tell if a turf is super cold unless you stepped on it, which for races sensitive to cold IE Unathi and Slime people that's mega dangerous. Considering you can see burny tile spots for heat...I see no reason cold should be changed.

**Comment by Fox-McCloud** [comment]
>Considering you can see burny tile spots for heat...I see no reason cold should be changed.

Uh, no you don't. You can see when a turf is on fire, and fire causes damage in a more direct way, by setting you, well, on fire.

You absolutely cannot see if an area is overeheated; you have to go by cues. Fire alarms, airlocks down, burnt tiles, and the likes. There's races specifically vulnerable to heat, as well, and they get along just fine by these indirect visual cues (and alerts they get shown for something being hot).

It'll be the same for cold; broken windows, areas exposed to space, and firelocks down/alarms going off should tell you "yeah, uh, maybe don't" instead of just forcing the airlock and rushing through it---and heat is generally far more deadly than cold is.

The behavior for temperature being something you infer is consistent; there is no cold counterpart to hotspots.

And again, I plan on adding iced turfs, specifically that are slippery. Having two icy turfs that do different things is confusing and inconsistent to players (uh, is this area exposed to space and cold or did someone just ice this turf?) ...and again, there's no way in hell that having LINDA *actually* ice a turf is going to be remotely fun to play with.

File: code/LINDA/LINDA_turf_tile.dm [hunk 1]
```diff
@@ -237,27 +230,19 @@
...
 	var/atmos_overlay = get_atmos_overlay_by_name(atmos_overlay_type)
 	if(atmos_overlay)
-		overlays -= atmos_overlay
-		mouse_opacity = MOUSE_OPACITY_ICON
+		vis_contents -= atmos_overlay
 
 	atmos_overlay = get_atmos_overlay_by_name(new_overlay_type)
 	if(atmos_overlay)
-		overlays += atmos_overlay
+		vis_contents += atmos_overlay
```
Comment (Fox-McCloud) [inline code/LINDA/LINDA_turf_tile.dm]: Pretty doggone clever of TG to do this.

**Comment by Mitchs98** [comment]
> You absolutely cannot see if an area is overeheated;

You can see burned tile marks after a fire on tiles as well as walls, was what I meant...

**Comment by Fox-McCloud** [comment]
>You can see burned tile marks after a fire on tiles as well as walls, was what I meant.

This is inferring something exists, rather than it being direct, which was exactly my point.

I might as well implement icy turfs now, in this same PR.

**Comment by Fox-McCloud** [comment]
Went ahead and made cryostylane cause icy turfs, so that icy turfs aren't completely gone.

**Comment by KasparoVy** [comment]
A bit sad about seeing the loss of the ice but it is what it is.

**Pull request merged**